code/datums/flock/flock.dm

```diff
@@ -690,6 +691,9 @@ var/flock_signal_unleashed = FALSE
 		for(var/keyPath in flock_conversion_paths)
 			if (!istype(O, keyPath))
 				continue
+			if (isnull(flock_conversion_paths[keyPath]))
+				qdel(O) //is this safe? probably?
```

Comment:
```suggestion
				qdel(O)
```
Could maybe just add back comment to info header comment for flock_conversion_paths, from #9069